_posts/2019-01-10-raja-0.7.0.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "RAJA 0.7.0 Released"
+---
+
+RAJA is a software abstraction that systematically encapsulates platform-specific code to enable applications to be portable across diverse hardware architectures without major source code disruption. The v0.7.0 release contains several major changes, new features, a variety of bug fixes, and expanded user documentation and accompanying example codes. Major changes include:
+
+- RAJA::forallN and RAJA::forall methods were marked deprecated in the v0.6.0 release and have been removed.
```

Comment:
@hauten -- you can (and should :)) mark inline code snippets like this as surrounded by backticks (one key left of the "1" key), that was we get the syntax highlighting. See https://guides.github.com/features/mastering-markdown/ for other markdown features.